docs/cli_reference.md

```diff
@@ -2,6 +2,13 @@
 
 The `llama` CLI tool helps you setup and use the Llama toolchain & agentic systems. It should be available on your path after installing the `llama-toolchain` package.
 
+### Subcommands 
+1. `download`: `llama` cli tools supports downloading the model from Meta or HuggingFace. 
+2. `model`: Shows additional properties of the llama models.
+3. `distribution`: A distribution is a set of REST API, this command allows you to manage (list, install, create, configure, start) distributions. You can read more about this here. 
```

Comment:
set of REST APIs (plural)

Reply:
where is "about this here" pointing to?

Reply:
I need the hyperlink to be created for the sub-headers first and then add the link in the next PR. Will take care of the fixes in the next PR.